src/server.js

```diff
@@ -286,10 +286,10 @@ function auth(data) {
 		}
 	} else {
 		client = manager.findClient(data.user, data.token);
-		var signedIn = data.token && data.token === client.config.token;
+		var signedIn = data.token && client && data.token === client.config.token;
```

Comment:
You actually removed this `client` check here: https://github.com/thelounge/lounge/commit/cf64cb04c493041756f1f3639f5b8d35dff21b27#diff-e6a5b42b2f7a26c840607370aed5301a